<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>1323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="_cam_presection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>616</width>
      <height>514</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>pre-section cam</string>
    </property>
   </widget>
   <widget class="QLabel" name="_cam_postsection">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>616</width>
      <height>514</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>post-section cam</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="btn_StartCams">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start cams</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_StopCams">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop cams</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_TrackSpeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Track tape speed</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_SaveVideos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save videos</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_StartCut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start cut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_StopCut">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop cut</string>
     </property>
    </widget>
   </widget>
   <widget class="PlotWidget" name="pg_sync" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>616</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="pg_phasediff" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>890</y>
      <width>616</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>920</y>
      <width>521</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Leica</string>
    </property>
    <widget class="QSpinBox" name="sbx_retractionSpeed">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6000</number>
     </property>
     <property name="value">
      <number>440</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_GetRetract">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>get retract speed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getNS">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>130</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>get S/N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getEW">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>get W/E</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="DisplayRetractSpeed">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="DisplayNS">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="DisplayEW">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_goNS">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>130</y>
       <width>91</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>go to S/N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_goEW">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>180</y>
       <width>91</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>go to W/E</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>183</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0-32 mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>132</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0-10 mm</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sbx_setNS">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sbx_setEW">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>180</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>42.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_skip">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>22</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>skip </string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>393</x>
       <y>26</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>sections</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SetRetract">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>set retract speed</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbx_NumSections">
     <property name="geometry">
      <rect>
       <x>322</x>
       <y>27</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>540</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Cutting vs slot</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Target cyle speed:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sbx_targetCycleSpeed">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.400000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tape speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>mm/sec</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sbx_tapeSpeed">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.400000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_synTS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>sync ts</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sbx_targetphase">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Target phase offset:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>175</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjustment factor:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_adjFactor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>177</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sbx_adjFactor">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_TapeStop">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>820</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TapeStart">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>820</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobtn_forward">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>790</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>forward</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobtn_backward">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>790</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>backward</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbx_tapeTension">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>870</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>870</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>set tension:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DisplayTension">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>870</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_GetTension">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>870</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>get tension</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1420</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
